modules/ROOT/pages/customize.adoc

```diff
@@ -0,0 +1,107 @@
+
+For detailed component information, refer to xref:acm-lightning-components.adoc[ACM Lightning Components] and
+the Salesforce Community Cloud https://help.salesforce.com/articleView?id=rss_component_reference_table.htm&type=5[Community Components, role=external, window=blank] documentation.
+
+Components that are typically used in *Home* pages include:
+
+* https://help.salesforce.com/articleView?id=rss_headline.htm&type=5[Headline, role=external, window=blank]
+* https://help.salesforce.com/articleView?id=rss_setup_nav_menu.htm&type=5[Navigation Menu, role=external, window=blank]: You can update *Max Items to Display (2-15)*,
+*Menu Item Alignment*, and use the *Navigation Menu* to display, add, remove, and customize options. *Home*
+is always present and is always the first item. The following properties can be customized:
+** *Name*
+** *Type*
+** *Page*
+** *Publicly available*
+* https://help.salesforce.com/articleView?id=rss_cms_collection.htm&type=5[CMS Collection, role=external, window=blank]
+* *API Carousel*: Displays APIs and API versions that are available to the community from Anypoint Exchange.
+* https://help.salesforce.com/articleView?id=community_builder_rich_text_component.htm&type=5[Rich text editor, role=external, window=blank]
+* https://help.salesforce.com/articleView?id=rss_html_editor.htm&type=5[HTML editor, role=external, window=blank]
+
+Components that are typically used in *API details* pages include:
+
+* *API Header*: Specifies the API name and description.
+* *Tabs*: Contains the following components: *API Documentation Viewer*, which displays API Exchange information,
+*API Console*, which renders information from the API RAML specification, and *Feed*, which displays comments,
+discussions, and questions.
+* *API Access Requestor*: Registers a client application and requests access to the API.
+
+Components that are typically used in *My Application* pages include:
+
+* *Application Listing*: Lists all client applications that the member has registered and can access.
+
+To reorder items on a page, simply drag and drop them.
+
+== Theme
+
+Select the *Theme* paintbrush icon in the top left navigation to access the following general branding parameters:
+
+* *Colors*: These settings include the site's *General* color scheme used throughout the community,
+including paragraph, heading, and button text.
+* *Images*: Specify the site's *Company Logo* and the login page's *Background Image*.
+* *Fonts*: Specify the site's fonts, which include a *Primary Font* and *Header Fonts*. You can also set the
+site's *Text Case*.
+* https://help.salesforce.com/articleView?id=community_builder_theme_adjust.htm&type=5[Theme Settings, role=external, window=blank]
+
+You can also edit CSS rules to adjust your community styling by selecting the *Theme* drop-down list arrow
+located next to the *Theme* title and selecting *Edit CSS*.
```

Comment:
"...selecting the *Theme* list located next to the *Theme* title

Reply:
Fixed.